Code/Tools/AssetProcessor/native/utilities/ApplicationManager.cpp

```diff
@@ -582,14 +582,14 @@ ApplicationManager::BeforeRunStatus ApplicationManager::BeforeRun()
         return ApplicationManager::BeforeRunStatus::Status_Failure;
     }
 
-    // enable stats capture from this point on
-    AssetProcessor::StatsCapture::Initialize();
-
     return ApplicationManager::BeforeRunStatus::Status_Success;
 }
 
 bool ApplicationManager::Activate()
 {
+    // enable stats capture from this point on
+    AssetProcessor::StatsCapture::Initialize();
+
```

Comment:
Is it safe to move this later?  There weren't any stats being recorded before this point I guess?

Reply:
I traced the code and ran the program. It seems that there is no stat being recorded before the new location. I also asked Nick for a double check.